tenants/ai-example/single-model-serving-vllm/test/granite-test.ipynb

```diff
@@ -105,7 +105,9 @@
     "        \"length_penalty\": 1\n",
     "    }\n",
     "\n",
-    "    response = requests.post(infer_url, json=json_data, verify=False)\n",
+    "    # If using RHOAI 2.13 or new, set `verify=True`\n",
```

Comment:
I assume that means these test won't pass if they are using an older version.

Do we want to add something to the failed test message that just says "Hey if this is older than 2.13 than this test is expected to fail"?